content/en/core/overview/translations.md

```diff
@@ -22,6 +22,7 @@ New languages must be added and configured in several places:
 - *In GitHub*
   - Create a new `messages-XX.properties` file in the [`api/resources/translations`](https://github.com/medic/cht-core/tree/master/api/resources/translations) folder, replacing XX with the 2 or 3 letter language code.
   - Add the language to the [`LOCAL_NAME_MAP` in api](https://github.com/medic/cht-core/blob/master/api/src/translations.js#L8). Use the language code for the key, and the local name followed by the English name for the language in brackets, eg: "fr: 'Français (French)'".
+  - Add an object to the `languages` array in `app_settings.json` with the `locale` and `enabled` properties representing respectively the 2 or 3 letter language code and whether that language should be enabled
```

Comment:
So, I think this docs page is focused on adding a new translation file _to the cht-core codebase_.  This is not what a partner should read when they are trying to setup the config _for their instance_.  

Maybe we should move this documentation to this page: https://docs.communityhealthtoolkit.org/apps/reference/translations/

The first paragraph on that page says:

> In the app_settings.json file the default language for the application is set, along with a separate default language for outgoing messages that are sent via SMS.

Right after that seems like a good place to add some more details...

Reply:
Good call, I agree this is a better place to document this